plugins/shopify/access_key.go

```diff
@@ -0,0 +1,52 @@
+package shopify
+
+import (
+	"context"
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func AccessKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:    credname.CLIToken,
+		DocsURL: sdk.URL("https://shopify.dev/docs/themes/tools/cli"),
+		//ManagementURL: sdk.URL("https://admin.shopify.com/store/{YOUR_STORE_ID}/apps/theme-kit-access"), // Can't support unless URL supports variable URLs
```

Comment:
Hmm, I see no reason why you can't just uncomment this part and leave the url as is: `https://admin.shopify.com/store/{YOUR_STORE_ID}/apps/theme-kit-access`. Obviously this isn't clickable but it will direct users towards where they could obtain the auth token, which is the goal of this field.